### ISSUE TITLE
Support for ARM64

### ISSUE THREAD

**Issue by airmonitor** [kubecost/cost-analyzer-helm-chart#652]
Hello.

As the ARM CPU architecture is getting more traction, for example AWS EKS based on ARM CPU was released can you please provide helm chart with updated container images which were compiled for various cpu architectures, arm64, x86_64?

Here is an example code how to build for multiple architectures in gitlab.

https://stackoverflow.com/questions/61430005/gitlab-ci-cd-building-multiarch-docker-images

**Comment by dwbrown2** [comment]
Hi Tomasz! Thanks for the request. Have you seen any helm chart examples that you feel have done this well? If so, we'll investigate and follow up.

**Comment by airmonitor** [comment]
Hello.

Sure :)

For example, influxdata/influxdb works out of the box.


```
helm show chart influxdata/influxdb                                                                                   
apiVersion: v1
appVersion: 1.8.0
description: Scalable datastore for metrics, events, and real-time analytics.
home: https://www.influxdata.com/time-series-platform/influxdb/
keywords:
- influxdb
- database
- timeseries
- influxdata
maintainers:
- email: rawkode@influxdata.com
  name: rawkode
- email: giacomo@influxdata.com
  name: gitirabassi
- email: urakiny@gmail.com
  name: aisuko
- email: naseem@transit.app
  name: naseemkullah
name: influxdb
sources:
- https://github.com/influxdata/influxdb
version: 4.8.9


```

**Comment by lordz-md** [comment]
Confronted with this today. We tried to deploy a AWS EKS cluster based on AWS ARM based Graviton processors. We get standard_init_linux.go:219: exec user process caused: exec format error and the pod enters in a CrashLoopBackOff

**Comment by mbolt35** [comment]
Thanks for the input. We do this currently for our network-costs pods, but we still need to revisit our build process for each of the other images we ship with. We'll update you here when we can. Thanks again

**Comment by dwbrown2** [comment]
@lordz-md do you already have Prom + kube-state-metrics running on your ARM machine? If so, we may need to just do this for our core images.

**Comment by lordz-md** [comment]
@dwbrown2 I just checked and prometheus and metrics-server are running.

**Comment by lordz-md** [comment]
@dwbrown2 my apologies kube-state-metrics is also failing with standard_init_linux.go:219: exec user process caused: exec format error.

**Comment by dwbrown2** [comment]
For reference it looks like ksm just added arm64 support: https://github.com/kubernetes/kube-state-metrics/issues/1340

**Comment by dwbrown2** [comment]
@AjayTripathy given that ksm now has support, it looks like it's just building the core kubecost pod with arm64 compatibility.

**Comment by dwbrown2** [comment]
All kubecost containers should now support running on arm-based machines. Please reach out if you have any questions or issues!

**Comment by jasonumiker** [comment]
I just deployed this chart to Graviton Instances on AWS (ARM64) and both the cost-model and cost-analyzer-frontend failed to start.

It works fine on the x64 instance types.

**Comment by dwbrown2** [comment]
A number of PRs now merged! @AjayTripathy expected to be in release this coming Tuesday?

**Comment by AjayTripathy** [comment]
This is expected in the release!

**Comment by kbrwn** [comment]
@airmonitor Support for ARM64 has shipped! Please try out kubecost v1.88.1. If you have any issues feel free to e-mail support@kubecost.com or join our slack at kubecost.slack.com.
